pkg/config/setup/config.go

```diff
@@ -893,6 +893,9 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("remote_updates", false)
 	config.BindEnvAndSetDefault("installer.registry.url", "")
 	config.BindEnvAndSetDefault("installer.registry.auth", "")
+
+	// Data Jobs Monitoring config
+	config.BindEnvAndSetDefault("djm_config.enabled", false)
```

Comment:
More of a question here, could you help confirm here: by default, this will allow the agent to fetch the configuration from env variable DD_DJM_CONFIG_ENABLED as well, is that right? Thanks